addons/web/static/tests/mobile/views/calendar_view_tests.js

```diff
@@ -25,6 +26,36 @@ QUnit.module("Views", ({ beforeEach }) => {
         setupViewRegistries();
 
         serverData = {
+            actions: {
+                1: {
+                    id: 1,
+                    name: "Event Action",
+                    res_model: "event",
+                    type: "ir.actions.act_window",
+                    views: [
+                        [false, "calendar"],
+                        [false, "kanban"]
```

Comment:
```suggestion
                        [false, "kanban"],
```
:slightly_smiling_face: